src/pages/register-a-beacon/check-beacon-details.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FunctionComponent } from "react";
+import { Grid } from "../../components/Grid";
+import { InsetText } from "../../components/InsetText";
+import { Layout } from "../../components/Layout";
+import { Button, BackButton } from "../../components/Button";
+import {
+  Form,
+  FormFieldset,
+  Input,
+  FormLegend,
+  FormLabel,
+  FormGroup,
+  Select,
+  SelectOption,
+  FormHint,
+} from "../../components/Form";
+import { Details } from "../../components/Details";
+import { A } from "../../components/Typography";
+
+const CheckBeaconDetails: FunctionComponent = () => (
+  <>
+    <Layout navigation={<BackButton href="/intent" />}>
+      <Grid
+        mainContent={
+          <>
+            <Form action="/register-a-beacon/check-beacon-summary">
+              <FormFieldset>
+                <FormLegend>Check beacon details</FormLegend>
+                <InsetText>
+                  The details of your beacon must be checked to ensure they have
+                  a UK encoding and if they are already registered with this
+                  service.
+                </InsetText>
+
+                <BeaconManufacturerSelect />
+
+                <BeaconModelSelect />
+
+                <BeaconHexIdInput />
+              </FormFieldset>
+              <Button buttonText="Submit" />
+            </Form>
+            <IfYouNeedHelp />
+          </>
+        }
+      />
+    </Layout>
+  </>
+);
+
+const BeaconManufacturerSelect: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="beaconManufacturer">
+      Select your beacon manufacturer
+    </FormLabel>
+    <Select name="beaconManufacturer" defaultValue="default">
+      <option hidden disabled value="default">
+        Beacon manufacturer
+      </option>
+      <SelectOption value="Raleigh">Raleigh</SelectOption>
+      <SelectOption value="Giant">Giant</SelectOption>
+      <SelectOption value="Trek">Trek</SelectOption>
+    </Select>
+  </FormGroup>
+);
+
+const BeaconModelSelect: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="beaconModel">Select your beacon model</FormLabel>
+    <Select name="beaconModel" defaultValue="default">
+      <option hidden disabled value="default">
+        Beacon model
+      </option>
+      <SelectOption value="Chopper">Chopper</SelectOption>
+      <SelectOption value="TCR">TCR</SelectOption>
+      <SelectOption value="Madone">Madone</SelectOption>
+    </Select>
+  </FormGroup>
+);
+
+const BeaconHexIdInput: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="beaconHexId">
+      Enter the 15 digit beacon HEX ID
+    </FormLabel>
+    <FormHint id="beaconHexId">
+      This will be on your beacon. It must be 15 characters long and use
+      characters 0-9, A-F
+    </FormHint>
+    <Input name="beaconHexId" spellCheck={false} />
+    <br />
+    <br />
```

Comment:
I would generally steer clear of adding breaks into html like this if you can.  Maybe have a look what the GOVUK design system looks like for a similar component to see if it's way off - I don't think the wireframes are true to the GOVUK design system in some cases.

If you need the padding, I would just add a css class and add the padding on there.  You might be able to do this by using the `FormGroup` component as that wraps a `div` element so you could put the styling on there